kolibri/plugins/coach/assets/src/composables/useFetchContentNode.js

```diff
@@ -0,0 +1,38 @@
+import { ref } from 'vue';
+import ContentNodeResource from 'kolibri-common/apiResources/ContentNodeResource';
+
+export default function useFetchContentNode(contentId) {
```

Comment:
Are there plans to use this further in other parts of coach?

Given it is only used in one place currently, I'm wondering if it'd be worth just doing the `ContentNodeResource.fetchModel` into the `PreviewSelectedResource` component where the individual node is needed.

If there are plans to use it elsewhere, then I'm thinking that we might want to ensure this is more configurable and generic. For example, the `getParams` w/ `no_available_filtering` is hard-coded, but what if we want to fetch a single content node w/ different params?

Reply:
May be in future we might need , but also we shall need this composable for the preview exercises